pkl-core/src/test/files/LanguageSnippetTests/input/api/pcfRenderer1.pkl

```diff
@@ -15,21 +15,21 @@ parrot {
   new {
     // multiline string that is a property within an element
     lastName = """
-      Flinstone
+      Flintstone
 
-      Flinstone
+      Flintstone
       """
     """
-    Flinstone
+    Flintstone
 
-    Flinstone
+    Flintstone
     """
   }
   // multiline string that is an element
   """
-  Flistone
+  Flintstone
```

Comment:
Similarly, there wasn't any particularly obvious reason to use `Flistone` instead (notably different from the above or the expected spelling)